python/GafferUI/CompoundEditor.py

```diff
@@ -52,52 +52,88 @@
 
 class CompoundEditor( GafferUI.Editor ) :
 
+	class _DetachedPanel( GafferUI.Window ) :
```

Comment:
Ahh yeah, the message may need updating. Its nested so it can be accessed from tests. The other way would be to add a protected factory method for panels on to `CompoundEditor`.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3167/commits/4361ca224e657be1db373d6d0fe91d18a252c235, will address messages when we have our squashing session after moving to Gaffer programmatic drag...